docs/medical-api/more-info/medical-record-summary.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Medical Record Summary"
+description: "Consolidate Medical Records Into a Single Document"
+---
+
+This guide will teach you how to generate a Medical Record Summary, which condenses all of patients' medical records into a single document.
+
+To do this, Metriport extracts FHIR resources across all available CDA documents for a patient and de-duplicates them, resulting in a set of unique medical records, which are then written into HTML and PDF files, making them extremely easy to navigate.
```

Comment:
nit: run on sentance

"To do this, Metriport extracts FHIR resources across all available C-CDA documents for a patient and de-duplicates them, resulting in a set of unique medical records. These are then rendered as HTML or PDF files, making them extremely easy to navigate."